toolchain/parse/testdata/packages/import/fail_after_decl.carbon

```diff
@@ -0,0 +1,25 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+
+fn A();
+
+// CHECK:STDERR: fail_after_decl.carbon:[[@LINE+3]]:8: ERROR: `import` directives must come after the `package` directive (if present) and before any other entities in the file.
+// CHECK:STDERR: import B api;
+// CHECK:STDERR:        ^
+import B api;
```

Comment:
I think it'd be best to drop the `api` here so this is a valid `import` declaration other than its location. (I assume this is a copy-paste and not intended?)

Reply:
Yes, copy-paste, fixing